features/origin-private-file-system.yml

```diff
@@ -0,0 +1,6 @@
+name: Origin private file system
+description: The `navigator.storage.getDirectory()` method returns a `FileSystemDirectoryHandle` that is restricted to a specific origin and invisible to the user's actual file system for faster file-based applications, such as SQLite databases.
+spec: https://fs.spec.whatwg.org/#origin-private-file-system
+compat_features:
+  - api.StorageManager.getDirectory
+  - api.FileSystemFileHandle.createSyncAccessHandle
```

Comment:
Is there a principle for which parts of the API surface are counted as this API? Everything that shipped in the same versions of browsers as the entry point would be a good start, but is it just this one method?

Reply:
Mostly, I just don't know this API well enough to be a good judge. I took a conservative approach here and put only those things that looked (to my inexperienced eye) _exclusive_ to OPFS. A less conservative approach would probably overlap significantly with [`file-and-directory-entries`](https://github.com/web-platform-dx/web-features/blob/main/features/file-and-directory-entries.yml).

I'm happy to do something reasonable here, I just didn't know where to start drawing the lines (or if we even ought to draw such lines).

Reply:
`getDirectory` is definitely the right entry-point to the OPFS feature. But I think we should also take a look at the `FileSystemHandle` interface and list what is supported where, and what, from it, OPFS really needs. Some of it is likely only needed for "real" file system access, and doesn't need to be listed here.

I think @tomayac would be a good person to help us list the exhaustive pieces of OPFS that developers really need to use the feature.